executor/executor.cc

```diff
@@ -105,6 +112,9 @@ struct thread_t {
 	pthread_t th;
 	// TODO(dvyukov): this assumes 64-bit kernel. This must be "kernel long" somehow.
 	uint64_t* cover_data;
+	// pointer to the size of coverage (stored as first word of memory)
```

Comment:
Stick with the existing comment style. Full line comments are full sentences -- start with a capital letter and end with a dot.